Makefile

```diff
@@ -379,7 +379,7 @@ govet:
     ./tools/...
 
 ineffassign:
-	@$(ECHO_CHECK) $(GOFILES)
+	@$(ECHO_CHECK) ineffassign
```

Comment:
You have a type in the 1st commit message - s/ineffassing/ineffassign/

Reply:
> Only one nit. The rest looks good.
> 
> You mentioned some false positives reported by prealloc on the weekly meeting. I guess that's not the case anymore? :)

No that's not the case anymore. I ran `prealloc` with the `-forloops` flag (which is disabled by default) and this lead to some cases being reported where pre-allocation doesn't make sense.

Reply:
> You have a type in the 1st commit message - s/ineffassing/ineffassign/

Thanks, will fix it.

Reply:
FWIW, I moved the preliminary cleanup commits (this change being among those) to a separate PR #10823 so it could already be merged before we decide on whether we want to enable `prealloc` or not.